.github/workflows/jacoco.yml

```diff
@@ -0,0 +1,51 @@
+name: Coverage
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Set up JDK 11 for Autogen
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 11
+          cache: 'maven'
+      - uses: actions/cache@v4
+        with:
+          path: ~/.m2/repository
+          key: shardingsphere-maven-third-party-e2e-cache-${{ github.sha }}
+          restore-keys: |
+            shardingsphere-maven-third-party-e2e-cache-
+            shardingsphere-maven-third-party-
+
+      - name: Build Maven Env for Autogen
```

Comment:
maybe `Install Project` is better ?

Reply:
I agree with that. Changed the step name.